loops-and-arrays.js

```diff
@@ -22,15 +24,22 @@ const names = ["Henk", "Piet", "Fred", "Joop"];
 // ==========================================
 
 const numbers = [2, 4, 5, 29, 38];
+    console.log(numbers)
 
 // Verwachtte uitkomsten:
 // Vóór het script zie je de originele waardes:
 // console.log(numbers) geeft: [2, 4, 5, 29, 38];
 // Na jouw script zie je de aangepaste waardes:
 // console.log(numbers) geeft: [ 4, 12, 10, 87, 76 ];
 
-
-
+for (let i = 0; i < numbers.length; i++) {
```

Comment:
Hier eigenlijk hetzelfde, je past ze aan in de for loop, maar niet de const "numbers" array zelf. 
In de uitwerkingen leggen ze het zo uit:
    // als je alleen de uitkomst logt (zoals hieronder) zou je de array niet veranderen, en dat wil je juist wel!
    // Dit is dus niet de juiste aanpak:
    // console.log(numbers[i] * 3);

Reply:
Verder zag ik ook jouw comment bij mij over deze. Ik snapte het zelf ook even niet hoe dit nu kon! In de uitwerkingen geeft het antwoord namelijk ook [ 4, 12, 10, 87, 76 ] - en dat klopt dus eigenlijk niet!
Maar na een tijdje zoeken zag ik het verschil.

for (let i = 0; i < numbers.length; i++) {
    if (numbers[i] % 2 === 0) {
        numbers[i] = numbers[i] * 2;
    } else {
        numbers[i] = numbers[i] * 3;
    }
}
console.log(numbers);   >>>> geeft [ 4, 8, 15, 87, 76 ]

&&

for (let i = 0; i < numbers.length; i++) {
    if (i % 2 === 0) {
        numbers[i] = numbers[i] * 2;
    }
    if (i % 2 !== 0) {
        numbers[i] = numbers[i] * 3;
    }

console.log(numbers);   >>>> geeft [ 4, 12, 10, 87, 76 ]

Als je goed kijkt zie je het verschil! Is wel een puzzel hoor ;)